index.html

```diff
@@ -59,5 +60,267 @@
+                            <p class="item-font  text-success m-0">From &#8377;79</p>
+                            <p class="item-font text-secondary m-0 ">Lowest Price Ever</p>
+                        </a>
+
+                        <a href="#"
+                            class="d-flex flex-column align-items-center justify-content-start gap-1 text-decoration-none"
+                            role="button">
+                            <div class="product-image">
+                                <img src="assets/product/lava.jpeg" alt="lava mobile" class="  object-fit-contain w-100 h-100" loading="lazy">
+                            </div>
+                            <p class="item-font  text-dark m-0 mt-auto fw-medium">LAVA A1500</p>
+                            <p class="item-font  text-success m-0">Flat &#8377;830 Off</p>
+                            <p class="item-font text-secondary m-0 ">Now &#8377;999</p>
+                        </a>
+                    </div>
+
+                    <button
+                        class=" arrow-btn btn btn-light arrow-btn top-0 bottom-0 end-0 my-auto rounded-0 px-2 h-25 fs-2 position-absolute">
+                        >
+                    </button>
+                </div>
+
+                <a href="#" class="bg-white d-none d-lg-block p-1" role="button">
+                    <img src="assets/product/offcard.png" alt="offer-card" class="h-100 item-offer" loading="lazy">
+                </a>
+            </div>
+
+            <!-- second-feed-section -->
+            <div class="products-item container p-0 px-md-0">
+                <div class="product-item-section1 container my-2 d-flex gap-2 p-0">
+                    <div
+                        class="product-item-section2 flex-grow-1 bg-white d-flex flex-column justify-content-start position-relative overflow-x-auto">
+
+                        <div class="deals d-flex align-items-center justify-content-between border-bottom px-3 ">
+                            <p class="d-flex flex-column m-0">
+                                <span class="fs-4">Books And music Instruments</span>
+                                <span class="text-secondary">Shop Now!</span>
+                            </p>
+                            <button class="d-none d-sm-block btn rounded-1 btn-primary fw-medium px-4 py-2">
+                                VIEW ALL
+                            </button>
+                        </div>
+
+                        <div class="product-section-main d-flex gap-4 p-3 flex-grow-1">
+                            <a href="#"
+                                class="d-flex flex-column align-items-center justify-content-start gap-1 text-decoration-none"
+                                role="button">
+                                <div class="product-image">
+                                    <img src="assets/product/harmonium.jpeg" alt="harmonium" class="  object-fit-contain w-100 h-100" loading="lazy">
```

Comment:
We can remove this extra space. (applicable for other occurances)

```suggestion
                                    <img src="assets/product/harmonium.jpeg" alt="harmonium" class="object-fit-contain w-100 h-100" loading="lazy">
```